ui/packages/tidb-dashboard-lib/src/apps/Monitoring/context/index.ts

```diff
@@ -18,7 +24,10 @@ export interface IMonitoringDataSource {
 
 export interface IMonitoringContext {
   ds: IMonitoringDataSource
-  metricsQueries: any
+  cfg: {
+    metricsQueries: any
+    clusterType: ClusterType
```

Comment:
How about `promAddrConfigurable: boolean`?

Reply:
The prometheus data on cloud only exist for 1 day, so we have to limit the TimeRangePicker to help user select appropriate time range. Since we also need to customize the time range picker on cloud monitoring page, the name `promAddrConfigurabale` might be inappropriate.

Reply:
I got you.